src/Sulu/Bundle/ContactBundle/Resources/public/js/components/accounts/components/form/main.js

```diff
@@ -387,6 +389,16 @@ define(['config', 'widget-groups'], function(Config, WidgetGroups) {
                 this.numberOfBankAccounts--;
                 this.updateBankAccountAddIcon(this.numberOfBankAccounts);
             }, this);
+
+            this.sandbox.on('sulu.router.navigate', this.cleanUp.bind(this));
+        },
+
+        /**
+         * Does some cleanup with aura components
+         */
+        cleanUp: function() {
+            // stop contact form before leaving
+            this.sandbox.stop(constants.editFormSelector);
```

Comment:
Why is this necessary at all? Something like this should already be executed in the on the navigate event, maybe the structure of the dom elements is wrong, and this is just fixing symptoms?